### DISCUSSION TITLE
Echo/Websocket Support for Notification

### DISCUSSION THREAD

**Discussion by RhysLees** [laravel/nova-issues#4196] [Ideas & Feature Requests]
I think it could be a good option to support real-time notifications. Often with the current implementation of polling for notifications, if a notification is just sent out you may have to wait a while to receive the notification if you just missed the poll. The issue with increasing the poll frequency is that if you have a lot of users, your server will be bombarded with requests.

Adding the option to use echo/WebSockets for notifications would limit requests and also would allow instant notification which improves user experience.

Of course, this would have to be an optional feature as it would otherwise gate keep users from using notifications that don't have access/means to use WebSockets but it would be great as an optional feature.

**Comment by ziming** [comment]
pusher support would be nice!

**Reply by RhysLees** [reply]
> pusher support would be nice!

We would just need laravel echo support, that would allow us to use multiple services
